Demo/Demo/SwiftUIComponents/PayPalWeb/PayPalWebButtonsView.swift

```diff
@@ -29,17 +28,22 @@ struct PayPalTransactionView: View {
                     .stroke(.gray, lineWidth: 2)
                     .padding(5)
             )
-            PayPalWebApprovalResultView(paypalWebViewModel: paypalWebViewModel)
-            if paypalWebViewModel.state.checkoutResult != nil {
+
```

Comment:
what would be a case where it's success and result is nil? Something went wrong on API side?

Reply:
Yeah, we update the `checkoutResult` based on what we get back in the `didFinishWithResult` delegate method, so if we hit another delegate we would not have a `checkoutResult` and thus not be allowed to move to the `PayPalWebTransactionView`

Reply:
oh I c,  success state is used for different results.